src/renderer/components/+welcome/welcome.scss

```diff
@@ -7,7 +7,7 @@
   text-align: center;
   width: 100%;
   height: 100%;
-  z-index: 1;
+  z-index: var(--z-index-base);
```

Comment:
What means base? Base should be `z-index: 0` which is by default for all elements?

Reply:
Default value for all elements is `z-index: auto`. Created new set of variables. Now `--z-index-base: 0`.